demo/config.yaml

```diff
@@ -21,3 +21,7 @@ lcr-modules:
         inputs:
             sample_bam: "data/{sample_id}.bam"
             sample_bai: "data/{sample_id}.bam.bai"
+
+    varscan:
+        inputs:
+            sample_bam: "data/{sample_id}.bam"
```

Comment:
Why is the .bai file not an input?

Reply:
I'm going to add that as an input, right now I'm assuming that the .bam and .bam.bai files are generated by the pipeline and I used 
```
op.relative_symlink(input.bam + ".bai", output.bam + ".bai")
```
in the `_varscan_input_bam`

Reply:
It's better to leave .bai as an input because then snakemake can raise errors early if they're missing.